src/SFA.DAS.DATA.MANAGEMENT/Tables/ASDataPL_Acc_Accounts.sql

```diff
@@ -0,0 +1,16 @@
+﻿CREATE TABLE AsData_Pl.Acc_Account
```

Comment:
we may not need this as we already adding the ComtLevyStatus to the AsData_Pl.Acc_Account table